File: datalib/src/main/java/nl/acidcats/tumblrlikes/core/usecases/appsetup/TumblrBlogUseCase.java
package nl.acidcats.tumblrlikes.core.usecases.appsetup;

import rx.Observable;

/**
 * Created on 17/10/2018.
 */
public interface TumblrBlogUseCase {
    Observable<Boolean> setTumblrBlog(String tumblrBlog);

    Observable<String> getTumblrBlog();
}

File: datalib/src/main/java/nl/acidcats/tumblrlikes/core/usecases/appsetup/TumblrBlogUseCase.kt
package nl.acidcats.tumblrlikes.core.usecases.appsetup

import rx.Observable

/**
 * Created on 24/10/2018.
 */
interface TumblrBlogUseCase {
    fun setTumblrBlog(tumblrBlog: String): Observable<Boolean>

    fun getTumblrBlog(): Observable<String>
}